	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/fromesberg" target="_blank">fromesberg</a>
			<div class="markdown"><p>Hi Shaven, as of now, all we have done is get bacteria to propagate X and Y in its DNA, and we initially avoided putting X and Y into a gene.  What we are doing now is just that, so we can examine how the unnatural information is retrieved in the form of RNA and then protein.  This is where we will see the largest effects.  But what we know now, is that storing the increased information does not really effect the bacteria, which we are very excited about.  To be clear, in our paper, we do report that expressing the transporter proteins themselves (which is required to get X and Y into the cell) does slow growth a little.  However, Yorke Zhang in my lab has already found a way to eliminate this, as part of our ongoing efforts to optimize the system.  Hope this helps and thanks for the question.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/jjberg2" target="_blank">jjberg2</a>
			<div class="markdown"><p>I keyed in on this line in the NPR article</p>
<blockquote>
<p>&quot;This is embarrassing. We have really horrible names,&quot; Romesberg says. &quot;They are abbreviations for very complex chemical names.&quot; He explains that because his lab has made and investigated many possible molecules over the years, &quot;we couldn't give every one of them a cute little name like X or Y or alpha or beta â€” because we simply examined too many of them.&quot;</p>
</blockquote>
<p>and was curious just how many different molecules you've tried over the years. Where did other possible molecules fail that these ones succeeded?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/yorkezhang" target="_blank">yorkezhang</a>
			<div class="markdown"><p>We have tried around 300 compounds in vitro. They were just less successful than our current X and Y because their fidelity in replication in vitro was not as good. We have not tried other compounds in our in vivo system.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Epicus2011" target="_blank">Epicus2011</a>
			<div class="markdown"><p>I don't know anything about DNA, but wouldn't adding X or Y cause a frameshift mutation with the already existing DNA? Or am I missing something?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/AskMrScience" target="_blank">AskMrScience</a>
			<div class="markdown"><p>Yes, but only if you put them in the middle of a gene, and then only if you put them in <em>in addition</em> to what was already there, rather than replacing an existing base.  They'd probably be doing a simple swap, like this:</p>
<p>Original:</p>
<blockquote>
<p>GCA TTC AAG CTC</p>
</blockquote>
<p>New: </p>
<blockquote>
<p>GCA TYC AAX CTC</p>
</blockquote>
<p>What those middle two codons would translate to depends on what tRNAs the lab cooks up.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Decaf_Engineer" target="_blank">Decaf_Engineer</a>
			<div class="markdown"><blockquote>
<p>and we initially avoided putting X and Y into a gene.</p>
</blockquote>
<p>There are lots of locations on a strand of DNA that gets replicated, but don't code for something useful.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Secs13" target="_blank">Secs13</a>
			<div class="markdown"><p>He said they only inserted the nucleotides in non-coding regions, so that wouldn't be a problem!</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Prufrock451" target="_blank">Prufrock451</a>
			<div class="markdown"><p>So the increased information density doesn't require the organism to consume a lot more energy to survive? If you were to insert X and Y into a multicellular lifeform, would the incremental difference create a big change in caloric requirements? </p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/yorkezhang" target="_blank">yorkezhang</a>
			<div class="markdown"><p>The increased information density will put a metabolic burden on the cells, since it needs to express extra machinery utilize the unnatural base pair. The cells already grow a little slower than regular lab E. coli when we express the transporter that's used to get the unnatural triphosphates into the cell.</p></div>		</li>
					</ul>
		</ul>
		</ul>
	