	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/yorkezhang" target="_blank">yorkezhang</a>
			<div class="markdown"><p>There are no H bonds between X and Y; they pair via hydrophobic interactions and shape complementarity. We have a paper published where we collaborated with the Marx lab to examine the structure of double helix DNA containing X and Y, as well as the structure of X and Y inside the active site of DNA polymerase.(<a href="http://www.nature.com/nchembio/journal/v8/n7/full/nchembio.966.html" target="_blank">http://www.nature.com/nchembio/journal/v8/n7/full/nchembio.966.html</a>)</p>
<p>X and Y allow for specific primer binding. They have a melting temperature intermediate between an AT and GC base pair. We have incorporated these base pairs into ssRNA in vitro, but not dsRNA. In vivo incorporation via transcription is one of our major focuses right now.</p>
<p>We currently cannot directly sequence these new base pairs because we do not have the dideoxynucleotides corresponding to X and Y. The presence of the unnatural base pair causes a termination in sequencing and we can also detect the unnatural base pair by PCR using related unnatural analogs that contain a chemical moiety that allows for detection via gel electrophoresis.</p>
<p>The bacteria must be constantly supplied with the unnatural triphosphates to allow for continual propagation of the unnatural base pair. When unnatural triphosphates are depleted, the unnatural base pair (X:Y) predominantly mutates to a (T:A) base pair, although our sequencing data says other pairs are possible.</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/sodium_dodecyl" target="_blank">sodium_dodecyl</a>
			<div class="markdown"><blockquote>
<p>We currently cannot directly sequence these new base pairs because we do not have the dideoxynucleotides corresponding to X and Y</p>
</blockquote>
<p>This explains why you couldn't sequence using Sanger sequencing, but is there any chance you could get it to work with next-gen sequencing?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/yorkezhang" target="_blank">yorkezhang</a>
			<div class="markdown"><p>Using new sequencing technologies to directly sequence our unnatural base pairs is currently a work in progress.</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Ergaar" target="_blank">Ergaar</a>
			<div class="markdown"><p><del>As far as i know Next generation sequencing methods also need the nucleotides to sequence because they do sequencing by synthesis. They build a complimentary strand to the DNA that is to be sequenced and use different detection methods to see which nucleotide is built in.</del></p>
<p>I misread dideoxynucleotides</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/cge" target="_blank">cge</a>
			<div class="markdown"><blockquote>
<p>They have a melting temperature intermediate between an AT and GC base pair.</p>
</blockquote>
<p>It would be interesting to see these incorporated into a nearest-neighbor thermodynamic model, as some of the differences seem like they could affect stability when viewed at this level (or perhaps longer) rather than at the single bp level. Have you looked at the stability of, say, poly-X/poly-Y?</p>
<p>I had initially wondered about orthogonality and the possibility of stable X-N / Y-N interactions, but given the very distinct interactions, I suspect this might not be an issue.</p>
<p>Unfortunately, in my field, the problem with unnatural base pairs always lies in synthesis costs... if large-scale synthesis were affordable, they would increase sequence spaces for short toeholds and sticky ends significantly.</p></div>		</li>
					</ul>
		</ul>
	