	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/electric_ionland" target="_blank">electric_ionland</a>
			<div class="markdown"><blockquote>
<p>In addition, they factor in anticipated technological advances on the ground. For example, a geosynchronous satellite launched 20 years ago may only be able to relay low-resolution static images, while a satellite launched today can relay high-resolution video. So the old tech may not have failed, but has outlived it's usefulness.</p>
</blockquote>
<p>This is an important point right now with the rise of &quot;smallsats&quot; and satellite constellations. People are reconsidering building heavy (half a dozen tons) and expensive (a few hundred million dollars) satellites that last a long time (~10 years). It might be more sensible to make 5 smaller sats that &quot;only&quot; cost 20 million dollars a piece and are replaced every few years so that they always get top of the line hardware. Since you have several of them a failure is not as damaging to the project and you might be able to take higher risks.</p>
<p>Those are very important system engineering considerations that are a hot topic right now in space engineering. </p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/sciklops" target="_blank">sciklops</a>
			<div class="markdown"><p>We're at a very similar point in genetics too, with technology improving dramatically every year while costs drop equally dramatically. </p>
<p>So, if planning a massive scale project (i.e. the US Precision Medicine Initiative, aka. &quot;All of Us&quot;) where 1 million patients will be recruited... should they be genotyped ($)? Exome sequenced ($$)? Genome sequenced ($$$)? What's the inflection point that balances cost with useful information gathering, and doesn't end up leaving out critical genetic info for these patients? Do we assume that most of the valuable information will be coded in common variants (which allows for use of genotyping only - the cheapest option), or do we want the resource database to be useful for study of rare variants (requires exome or genome sequencing)? If we go all-in on whole-genome sequencing, do we know enough to make the extra effort and expense worth it, and can we even computationally handle the volume of data that is generated? (answer: no to both at this point) Should the resource and biobanking plan include archive of patient DNA samples with the intention of generating more comprehensive genomic data from a subset?</p>
<p>In the end, it's all compromise, even at the outset.</p>
<p>FYI in this example, the plan is to genotype all patients and biobank DNA. No stated plan for full sequencing of a subset, though the raw material will theoretically remain available.</p></div>		</li>
					</ul>
		</ul>
	