	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/monarc" target="_blank">monarc</a>
			<div class="markdown"><p>I'd argue that this is not analogous to compression. It's closer to how words can have different meanings based on context. I don't think I'd say that language is &quot;compressed&quot; for this reason.  </p>
<p>Alternative splicing is a cool idea to bring up, and it certainly lets the genome use its space more efficiently (and in that broad sense it's somewhat comparable to compression). Another way this happens is genetic regulation, which is spatial/temporal control of what molecules are present and in what forms/amounts. </p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/michiforjoy" target="_blank">michiforjoy</a>
			<div class="markdown"><p>Is that akin to the use of pointers in programming languages (e.g., C++)?  For example, suppose area X on gene 4 relates to eye color, but actually the DNA says in part &quot;go use the DNA found in area PQR on gene 3.&quot;  And then area Y on gene 7 relates to hair color, which says in part &quot;go use the DNA found in area QRS on gene 3.&quot;  In this example, area QR on gene 3 would be used by both eye color and hair color.</p>
<p>Is that how it works, or is that way off?  I've read things on places like 23andme that certain genetic analyses are only confirmed when the person is a particular race, so I was wondering if there are &quot;pointers&quot; within the DNA of some races that &quot;point&quot; to different gene areas for a trait.  A Caucasian person's DNA might say to look at area PQR on gene 3 for eye color, but a Chinese person's DNA might say to look at area FGH on gene 12.  </p>
<p>Is that at all how it works?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Mvarela150" target="_blank">Mvarela150</a>
			<div class="markdown"><p>An analogy for splicing is that you have a string of words which can be chosen in any combination to make a new sentence. The gene (string of words) may encode proteins with different functions (different sentences) depending on how it is alternatively spliced.</p>
<p>So for example say you have the string:
&quot;Batman is has can a batmobile batcave do the cha cha millionaire&quot;</p>
<p>If you cut it like this (splice):
&quot;Batman is \ has can / a \ batmobile batcave do the cha cha / millionaire&quot;</p>
<p>You can get the sentence
&quot;Batman is a millionaire&quot;</p>
<p>Another splicing might be
&quot;Batman \ is has / can \ a batmobile batcave / do the cha cha \ millionaire /&quot;</p>
<p>Which gives you
&quot;Batman can do the cha cha&quot;</p>
<p>Note the forward-slash and backslash are used here to surround the words being cut out.</p>
<p>Hope that makes some sense!</p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/lets_trade_pikmin" target="_blank">lets_trade_pikmin</a>
			<div class="markdown"><p>That is sort of how it works because a single protein expressed by a single gene can be reused to build many other proteins when combined with products of other genes. However, that's not the phenomenon he was referring to. Alternative splicing is actually using the same sequence of &quot;bits&quot; to code for multiple products by utilizing redundancies between those different sequences, which is also common in digital data compression.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Skepsis93" target="_blank">Skepsis93</a>
			<div class="markdown"><p>To piggyback off your answer since I'm late to the party, I want to share <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4b/Chromatin_Structures.png/1280px-Chromatin_Structures.png" target="_blank">this</a> image detailing the different possible structure formations of DNA. In this image there are 2 parts that are possibly analogous to data compression, or at least as close as it can get in nature. </p>
<p>First, look at the actively transcribed genes that follow the beads on a string model. These genes are not &quot;compressed&quot; and can be actively transcribed into proteins. Directly to the right of that image where it says &quot;add histone H1&quot; is the compressed version. Basically what's happening here is that peripheral proteins such as histones help the DNA fold itself up into a more compact form. Usually computer files are compressed for storage and to save space, but here the compression by the histone proteins is more to prevent transcription of inappropriate genes than to save space. </p>
<p>The second example is at the right end and I think it is more analogous to data compression of the two scenarios. To start, look at the images under the &quot;Active Chromosome During Interphase.&quot; This is a representation of what your DNA looks like when it is simply floating around in the nucleus. It's pretty loose and can float freely in the nucleus. If you want to transcribe the DNA for protein production, it needs to be in this form. Next, look directly to the right and you'll see a more compressed version and to the right of that is what most people think of as DNA, or chromosomes. A chromosome is the most compact version of DNA and is tightly held together by the chromatin, which consists of all of the molecules and proteins that work in the periphery of DNA as well as the DNA itself. This only happens when a cell is actively dividing and this happens so no data is lost in the transfer. </p>
<p>So basically a chromosome during cell replication is a flash drive carrying DNA in a .zip file.  </p>
<p>Edit: Disclaimer - my area is biology and not computer science. So I'm not sure how compression works from a computer standpoint, I just know a .zip file takes up less space on my computer and can't be directly accessed so that's where I'm drawing the analogies from.   </p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/pickled_dreams" target="_blank">pickled_dreams</a>
			<div class="markdown"><p>Again, data compression != physical compaction. Your explanation is very good but it doesn't really have anything to do with OP's question.</p>
<p>Data compression is basically representing a sequence of symbols using a shorter (&quot;compressed&quot;) sequence of symbols. It has nothing to do with the physical size of the medium used to store those symbols.</p>
<p>In other words, putting a hard drive in a trash compactor is NOT data compression ;)</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/Skepsis93" target="_blank">Skepsis93</a>
			<div class="markdown"><p>Yes, that's why I said it's as a analogous as we can get. The end result of compressing a file is that it is easier to transfer and takes up less space, which chromosomes accomplish, albeit by a different method. </p>
<p>There is some <a href="http://www.tritechresearch.com/shop/images/aachart.gif" target="_blank">redundancy</a> in the way the genetic code works, since it uses 3 letter variations of ATC &amp; G to identify amino acids. But, the only conceivable way to compress it further outside of splicing (which also doesn't fit the strict definition of compression) would be to reduce those down to 2 letter variations of ATC &amp; G but you can see from the chart that this would not be possible without losing vital information. Theoretically CT, GT, TC, CC, AC, GC, CG, and GG could all be shortened to two letters while keeping the rest 3 letters. But then you come across the problem of how does the cell know when to read 3 letters and when to read 2 letters and isn't seen in nature. </p></div>		</li>
					</ul>
		</ul>
		</ul>
	