	<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/IEEESpectrum" target="_blank">IEEESpectrum</a>
			<div class="markdown"><p>There could be: bacterial genomes are circular (rather than having disjoint chromosomes as humans do) so there is an opportunity to use the number of twists in a circular genome to encode something (and indeed bacteria selectively wind and unwind their genomes).  A scheme that uses topological properties of DNA would have to overcome challenges like figuring out how to shape DNA, how to read out its shape (current sequencers just give you the sequence), and the math of mapping bitstrings to shapes.  Seems hard but fun!</p>
<p>There is a lot of work already on using the DNA sequence (rather than shape) to encode information (see, e.g., <a href="https://spectrum.ieee.org/semiconductors/devices/exabytes-in-a-test-tube-the-case-for-dna-data-storage" target="_blank"><a href="https://spectrum.ieee.org/semiconductors/devices/exabytes-in-a-test-tube-the-case-for-dna-data-storage" target="_blank">https://spectrum.ieee.org/semiconductors/devices/exabytes-in-a-test-tube-the-case-for-dna-data-storage</a></a>).  One of our collaborators (Hanlee Ji at Stanford) is also developing methods for reading information out of DNA in a way protected from noise (e.g. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28934929" target="_blank"><a href="https://www.ncbi.nlm.nih.gov/pubmed/28934929" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/28934929</a></a>).</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/iorgfeflkd" target="_blank">iorgfeflkd</a>
			<div class="markdown"><p>Soooo I actually misread the AMA and thought it was about storing data in DNA rather than storing genetic data on hard drives.</p></div>		</li>
					</ul>
			<li class="reply">
			<a class="author" href="https://www.reddit.com/user/DomDeluisArmpitChild" target="_blank">DomDeluisArmpitChild</a>
			<div class="markdown"><p>Yes! But probably not in the way you're thinking. DNA is a really cool chemical, capable of forming all sorts of associations with itself. </p>
<p>Chromosomal conformation capture (3C) is an area of active research, both in how to do it effectively, and in what it reveals. The 3d associative structures of DNA tells us which sections of a genome associate with each other. What happens is that we'll see segments of chromosomes associate with each other at different locations in the nuclei.</p>
<p>Some of these associative loci are genes that tend to be activated together; by stringing regulation across the DNA itself, a cell can activate a handful of genes across chromosomes with a single regulatory mechanism. That's just one example, and chromosomal conformation is a /lot/ more complicated than simple regulation. For example, chromosomal associations change based on the development stage of the organism, the type of cell in question, and which stage of the cell cycle its in. </p>
<p>The human genome is incredibly complicated, so most of our research has been limited to model organisms, and I'm far from an expert.</p>
<p>There's a lot about DNA that we don't know, and 3C technology will help us understand it better.</p>
<p>Also, I don't think the technology the op has worked on is really related to the 3d shape of the chromosome. </p></div>		</li>
						<li class="reply">
			<a class="author" href="https://www.reddit.com/user/dampew" target="_blank">dampew</a>
			<div class="markdown"><p>You mean like epigenetic information?</p></div>		</li>
					<ul class="replies">
		<li class="reply">
			<a class="author" href="https://www.reddit.com/user/iorgfeflkd" target="_blank">iorgfeflkd</a>
			<div class="markdown"><p>Nope.</p></div>		</li>
					</ul>
		</ul>
	